core/src/test/scala/org/apache/spark/storage/DiskStoreSuite.scala

```diff
@@ -150,7 +151,7 @@ class DiskStoreSuite extends SparkFunSuite {
 
     assert(diskStore.getSize(blockId) === testData.length)
 
-    val diskData = Files.toByteArray(diskBlockManager.getFile(blockId.name))
+    val diskData = Files.readAllBytes(diskBlockManager.getFile(blockId.name).toPath)
```

Comment:
Which one is better,  `Files.readAllBytes` or `com.google.common.io.ByteStreams#toByteArray(java.io.InputStream)`?